src/content/insertable-streams/video-processing/js/webcodec-transform.js

```diff
@@ -42,6 +42,7 @@ class WebCodecTransform { // eslint-disable-line no-unused-vars
     }
     this.controller_ = controller;
     this.encoder_.encode(frame);
+    frame.close();
```

Comment:
use try / finally to ensure frame.close() is called even if there's some exception?

Reply:
SG